src/backend/distributed/transaction/transaction_management.c

```diff
@@ -297,7 +297,8 @@ CoordinatedTransactionCallback(XactEvent event, void *arg)
 			MemoryContext previousContext =
 				MemoryContextSwitchTo(CitusXactCallbackContext);
 
-			if (CurrentCoordinatedTransactionState == COORD_TRANS_PREPARED)
+			if (CurrentCoordinatedTransactionState == COORD_TRANS_PREPARED &&
+				!isManagementCommand)
```

Comment:
isManagementCommand -> IsManagementCommand

(usually start global variables with uppercase to easily distinguish them)